pages/cookbook/index.mdx

```diff
@@ -114,30 +114,216 @@ dump("The loop is over!");
 
 ## Map
 
+For description and basic usage see [`map<k, v>{:tact}` type in the Book](/book/maps).
+
+### How to emulate an array using a map
```

Comment:
We explain how to do it, but then at the end discourage this recipe and show how this is actually supposed to be done. To me it makes sense to remove this section and show the proper way immediately

Reply:
My idea was to showcase a transition between straighforward approach with manual management to the version with extension functions and such. Idk if we should disregard the former, though

Reply:
Oh, and I terribly want to rename that `Array` example into `Stack`, as push/pop are more of a stack thing. It's just JS confused those terms and mixed them together :)